test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,316 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and aten operators."""
+import torch
+from torch.testing._internal import (
+    common_device_type,
+    common_methods_invocations,
+    common_utils,
+)
+
+# Ops and the dtypes to be tested for consistency
+# TODO(justinchuby): We need a way to annotate symbolic functions with their supported dtypes
+ALLOWLIST_OP = {
+    "__getitem__": frozenset(["b8", "f16", "f32", "i16", "i32", "i64", "u8"]),
```

Comment:
Does lintrunner take care of the sorting based on key str?

Reply:
I don't think so. We will need to sort by hand